src/SamplesApp/UITests.Shared/Windows_UI_Xaml/xLoadTests/xLoad_FindName.xaml.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Windows_UI_Xaml.xLoadTests
+{
+	[SampleControlInfo("xLoad", "xLoad_FindName")]
```

Comment:
This attribute is deprecated and the new `[Sample]` should be used instead.

The default category is the namespace name and the default name for the sample is the class name.